source/reference/cutover-process.txt

```diff
@@ -46,7 +46,19 @@ Steps
         If ``lagTimeSeconds`` isn't close to ``0`` when the cutover
         starts, cutover might take a long time.
 
-      The following example returns the status of the synchronization process.
+      - When using the :ref:`c2c-start-verifier` check **both**
+        the ``verification.source`` and the
+        ``verification.destination`` return documents. The
+        ``lagTimeSeconds`` fields in both documents should be
```

Comment:
Can we also update the sample response https://deploy-preview-339--docs-cluster-to-cluster-sync.netlify.app/reference/cutover-process/#response to reflect this?
Right now `verification.source.lagTimeSeconds` is 16 and the `phase` is `initial hashing` in the sample.

Reply:
Fixed